openassessment/xblock/test/data/invalid_assessments.json

```diff
@@ -118,6 +118,23 @@
         "current_assessments": null,
         "is_released": false
     },
+    "remove_staff_mid_flight": {
+        "assessments": [
+            {
```

Comment:
From the acceptance criteria: `Once a problem is live, ensure that you cannot remove staff grading`. This test case is trying to change this (`assessments` and `current_assessments` represent the new and previous states, respectively)

ORA2 is already set up to disallow removing an assessment type after the problem is published (see the block here: https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/validation.py#L148), so we get this for free. This is just adding a test case to make the staff-specific behavior be explicitly tested. 

`remove_peer_mid_flight` does the same thing just below this, but for peer assessment.